DcShare/Controller/ShareViewController.swift

```diff
@@ -65,11 +66,16 @@ class ShareViewController: SLComposeServiceViewController {
         if dbHelper.currentDatabaseLocation == dbHelper.sharedDbFile {
             dcContext.logger = self.logger
             dcContext.openDatabase(dbFile: dbHelper.sharedDbFile)
-            selectedChatId = dcContext.getChatIdByContactId(contactId: Int(DC_CONTACT_ID_SELF))
-            if let chatId = selectedChatId {
-                selectedChat = dcContext.getChat(chatId: chatId)
+            isAccountConfigured = dcContext.isConfigured()
+            if isAccountConfigured {
+                isAccountConfigured = true
```

Comment:
`isAccountConfigured` is always `true` here.

Reply:
copy and paste error, thanks